res/img/math.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" standalone="yes"?><svg xmlns="http://www.w3.org/2000/svg" height="45" viewBox="0 0 69 45" width="69" style="zoom: 8;"><path d="M6,0 h57 c3.3137085,0 6,2.6862915 6,6 v33 c0,3.3137085 -2.6862915,6 -6,6 h-57 c-3.3137085,0 -6,-2.6862915 -6,-6 v-33 c0,-3.3137085 2.6862915,-6 6,-6 m23.3703704,11 c-.6518551,0 -1.1841955,.3860101 -1.5970371,1.158042 l-7.7244444,14.7006993 l-7.7896296,-14.7006993 c-.4128416,-.7720319 -.9560461,-1.158042 -1.6296297,-1.158042 c-.4780271,0 -.86913425,.1554763 -1.1733333,.4664336 c-.30419906,.3109572 -.4562963,.7130511 -.4562963,1.2062937 v19.7510489 c0,.4717973 .13580111,.8524461 .40740741,1.1419581 c.27160629,.2895119 .63555329,.4342657 1.09185189,.4342657 c.478027,0 .8474061,-.1393925 1.1081481,-.4181818 c.2607421,-.2787893 .3911111,-.6647994 .3911111,-1.158042 v-14.8293706 l6.4207408,11.8699301 c.4128415,.7505865 .9451819,1.1258741 1.597037,1.1258741 s1.1841955,-.3752876 1.597037,-1.1258741 l6.3881482,-11.9986014 v14.9580419 c0,.4932426 .130369,.8792527 .3911111,1.158042 c.260742,.2787893 .619257,.4181818 1.0755555,.4181818 c.4780271,0 .8528382,-.1393925 1.1244445,-.4181818 s.4074074,-.6647994 .4074074,-1.158042 v-19.7510489 c0,-.4932426 -.1520972,-.8953365 -.4562963,-1.2062937 c-.304199,-.3109573 -.6953062,-.4664336 -1.1733333,-.4664336 m18.1296296,17.8786797 l-7.3180195,-7.3180195 c-.5857864,-.5857865 -1.5355339,-.5857865 -2.1213203,0 c-.5857865,.5857864 -.5857865,1.5355339 0,2.1213203 l9.8994949,9.899495 c.3056756,.3056756 .7104567,.4518432 1.1109127,.438503 c.400456,.0133402 .8052371,-.1328274 1.1109127,-.438503 l9.899495,-9.899495 c.5857864,-.5857864 .5857864,-1.5355339 0,-2.1213203 c-.5857865,-.5857865 -1.535534,-.5857865 -2.1213204,0 l-7.4601551,7.4601551 v-16.5208153 c0,-.8284271 -.6715729,-1.5 -1.5,-1.5 s-1.5,.6715729 -1.5,1.5  z " visibility="visible" fill-rule="evenodd" fill="#d8d8d8"></path><rect stroke="#000000" x="5.875px" height="31.5px" y="7.5px" id="rect1" stroke-width="0" width="55.5px" fill="#d7d8d7" transform="" visibility="hidden"></rect><text stroke="none" style="outline-style:none;" id="text1" stroke-width="1px" x="48.625px" text-rendering="geometricPrecision" font-family="AmericanTypewriter-Bold" fill="#000000" font-size="27px" y="30.75px" transform="" text-anchor="middle" visibility="hidden">∑</text><text stroke="none" style="outline-style:none;" id="text2" stroke-width="1px" x="20.375px" text-rendering="geometricPrecision" font-family="AmericanTypewriter" y="37.125px" fill="#000000" font-size="40px" transform="" text-anchor="middle" visibility="hidden">$</text></svg>
```

Comment:
Is this right?  It looks like the markdown logo still.

Reply:
I based it on the Markdown logo for some consistency:
<img width="265" alt="Screenshot 2019-08-05 at 21 27 47" src="https://user-images.githubusercontent.com/7130759/62493011-e8958a00-b7c7-11e9-96a2-f182f1009d64.png">

Reply:
<img width="522" alt="Screenshot 2019-08-05 at 21 28 40" src="https://user-images.githubusercontent.com/7130759/62493047-006d0e00-b7c8-11e9-8a0f-1554add2fd3c.png">
(zoomed in)

Reply:
I mean [math.svg](https://github.com/matrix-org/matrix-react-sdk/blob/ffb596f0f73774a61c0e31575a5fa68258e96c1a/res/img/math.svg) specifically is the Markdown logo.  `math.png` is fine.

Reply:
sorry, you're entirely right. I have no idea how `markdown.svg` was made, so just opted for a png instead, but I created a `math.svg` file (by copying the markdown one) just so the resource existed (for whenever somebody could make an svg)

Reply:
just fixed this, thanks to `swedneck:permaweb.io`, so now the svg version is used instead (and the temporary png deleted)